compose-files/pleuston.yml

```diff
@@ -0,0 +1,17 @@
+version: '2.1'
+services:
+  pleuston:
+    image: oceanprotocol/pleuston:${OCEAN_VERSION:-stable}
+    networks:
+      backend:
+        ipv4_address: 172.15.0.18
+    ports:
+    - 3000:3000
+    depends_on:
+    - keeper-contracts
+    environment:
+      KEEPER_HOST: keeper-contracts
+      OCEAN_HOST: aquarius
```

Comment:
There is no KEEPER_HOST and OCEAN_HOST anymore, variables need to target localhost since browser is not in docker network. If variables are not set then they default to localhost anyway so these lines are not needed actually
NODE_HOST: localhost
AQUARIUS_HOST: localhost
BRIZO_HOST: localhost

Reply:
Ok I understand. So the browser connect directly to the components.